packages/ui/src/views/webhooks/webhook-list/repo-webhook-list-page.tsx

```diff
@@ -1,208 +1,48 @@
+  getFilterOptions,
+  getLayoutOptions,
+  getSortDirections,
+  getSortOptions
+} from '@views/repo/repo-list/filter-options'
 import { formatDistanceToNow } from 'date-fns'
 
 import { SandboxLayout } from '../../index'
 import { RepoWebhookList } from './repo-webhook-list'
 import { WebhookListProps } from './types'
 
-const BASIC_CONDITIONS: FilterCondition[] = [
-  { label: 'is', value: 'is' },
-  { label: 'is not', value: 'is_not' },
-  { label: 'is empty', value: 'is_empty' },
-  { label: 'is not empty', value: 'is_not_empty' }
-]
-
-const RANGE_CONDITIONS: FilterCondition[] = [
-  { label: 'is', value: 'is' },
-  { label: 'is before', value: 'is_before' },
-  { label: 'is after', value: 'is_after' },
-  { label: 'is between', value: 'is_between' },
-  { label: 'is empty', value: 'is_empty' },
-  { label: 'is not empty', value: 'is_not_empty' }
-]
-
-const TEXT_CONDITIONS: FilterCondition[] = [
-  { label: 'is', value: 'is' },
-  { label: 'is not', value: 'is_not' },
-  { label: 'contains', value: 'contains' },
-  { label: 'does not contain', value: 'does_not_contain' },
-  { label: 'starts with', value: 'starts_with' },
-  { label: 'ends with', value: 'ends_with' },
-  { label: 'is empty', value: 'is_empty' },
-  { label: 'is not empty', value: 'is_not_empty' }
-]
-
-const NUMBER_CONDITIONS: FilterCondition[] = [
-  { label: '=', value: 'equals' },
-  { label: '≠', value: 'not_equals' },
-  { label: '>', value: 'greater' },
-  { label: '<', value: 'less' },
-  { label: '≥', value: 'greater_equals' },
-  { label: '≤', value: 'less_equals' },
-  { label: 'Is empty', value: 'is_empty' },
-  { label: 'Is not empty', value: 'is_not_empty' }
-]
-
-const FILTER_OPTIONS: FilterOption[] = [
-  {
-    label: 'Type',
-    value: 'type',
-    type: 'checkbox',
-    conditions: BASIC_CONDITIONS,
-    options: [
-      { label: 'Enabled', value: 'enabled' },
-      { label: 'Disabled', value: 'disabled' }
-    ]
-  },
-  {
-    label: 'Created time',
-    value: 'created_time',
-    type: 'calendar',
-    conditions: RANGE_CONDITIONS
-  },
-  {
-    label: 'Name',
-    value: 'name',
-    type: 'text',
-    conditions: TEXT_CONDITIONS
-  },
-  {
-    label: 'Stars',
-    value: 'stars',
-    type: 'number',
-    conditions: NUMBER_CONDITIONS
-  }
-]
-
-const SORT_OPTIONS: SortOption[] = [
-  { label: 'Last updated', value: 'updated' },
-  { label: 'Title', value: 'title' }
-]
-
-const SORT_DIRECTIONS: SortDirection[] = [
-  { label: 'Ascending', value: 'asc' },
-  { label: 'Descending', value: 'desc' }
-]
-
 const LinkComponent = ({ to, children }: { to: string; children: React.ReactNode }) => <Link to={to}>{children}</Link>
 
 const RepoWebhookListPage: React.FC<WebhookListProps> = ({ useWebhookStore, useTranslationStore }) => {
   const { t } = useTranslationStore()
 
+  const FILTER_OPTIONS = getFilterOptions(t)
+  const SORT_OPTIONS = getSortOptions(t)
+  const SORT_DIRECTIONS = getSortDirections(t)
+  const LAYOUT_OPTIONS = getLayoutOptions(t)
```

Comment:
I don't think we need to pass `t` function down to all components. We can just call the `useTranslationStore` hook everywhere directly.

Reply:
This will be fixed in the next PR